### PR TITLE
PR for Bug Fix#1-30085299

### PR DESCRIPTION
**Pull request by thiyagu06**

### CODE REVIEW

File: API/code/restapi/src/main/java/org/panalpina/api/rest/controller/AirFreightController.java [hunk 1]
```diff
@@ -223,23 +229,32 @@ private String formatDate(String ETD) {
...
+		ObjectMapper objectMapper = new ObjectMapper();
+		JsonNode rootNode = objectMapper.readTree(json);
+		JsonNode flightInfoStatus = rootNode.get("FlightInfoStatusResult");
+		if (flightInfoStatus == null) {
+			LOGGER.debug("flight Info status is null");
+			return null;
+		}
+		JsonNode flights = flightInfoStatus.get("flights");
+		
+		if(flights == null) {
```
Comment (sah2ed) [inline API/code/restapi/src/main/java/org/panalpina/api/rest/controller/AirFreightController.java]: Please format whitespace consistently:
`if(flights == null) {` ==> `if (flights == null) {`

**Comment by sah2ed** [comment]
@thiyagu06 

I can no longer see your Google Maps API key changes. Did you drop it from the patch?

**Comment by sah2ed** [comment]
Also, please submit your updated patch to OR with the `git am >` typo fixed .

**Comment by thiyagu06** [comment]
I have posted in the forum that each non billing API key only one load is allowed per day. Pls check. So I haven't made any changes related to API Key

File: API/code/restapi/src/main/java/org/panalpina/api/rest/controller/AirFreightController.java (hunk 1 above)
Comment (thiyagu06) [reply API/code/restapi/src/main/java/org/panalpina/api/rest/controller/AirFreightController.java]: fixed

**Pull request merged**